addons/payment_ogone/views/payment_ogone_templates.xml

```diff
@@ -2,6 +2,31 @@
 <odoo>
 
     <template id="redirect_form">
+        <form t-att-action="api_url" method="post">
+            <input type="hidden" name="PSPID" t-att-value="PSPID"/>
+            <input type="hidden" name="ORDERID" t-att-value="ORDERID"/>
+            <input type="hidden" name="AMOUNT" t-att-value="AMOUNT"/>
+            <input type="hidden" name="CURRENCY" t-att-value="CURRENCY"/>
+            <input type="hidden" name="LANGUAGE" t-att-value="LANGUAGE"/>
+            <input type="hidden" name="EMAIL" t-att-value="EMAIL"/>
+            <input type="hidden" name="OWNERADDRESS" t-att-value="OWNERADDRESS"/>
+            <input type="hidden" name="OWNERZIP" t-att-value="OWNERZIP"/>
+            <input type="hidden" name="OWNERTOWN" t-att-value="OWNERTOWN"/>
+            <input type="hidden" name="OWNERCTY" t-att-value="OWNERCTY"/>
+            <input type="hidden" name="OWNERTELNO" t-att-value="OWNERTELNO"/>
+            <input type="hidden" name="OPERATION" t-att-value="OPERATION"/>
+            <input type="hidden" name="USERID" t-att-value="USERID"/>
+            <input type="hidden" name="ACCEPTURL" t-att-value="ACCEPTURL"/>
+            <input type="hidden" name="DECLINEURL" t-att-value="DECLINEURL"/>
+            <input type="hidden" name="EXCEPTIONURL" t-att-value="EXCEPTIONURL"/>
+            <input type="hidden" name="CANCELURL" t-att-value="CANCELURL"/>
+            <input type="hidden" name="ALIAS" t-att-value="ALIAS"/>
+            <input type="hidden" name="ALIASUSAGE" t-att-value="ALIASUSAGE"/>
+            <input type="hidden" name="SHASIGN" t-att-value="SHASIGN"/>
+        </form>
+    </template>
+
+    <template id="redirect_form_validation">
```

Comment:
@tbe-odoo Note that this change will require you to update the view arch in the prod accordingly!